src/flavor/object-flavor.cpp

```diff
@@ -92,18 +92,19 @@ static bool object_easy_know(int i)
 
 /*!
  * @brief 各種語彙からランダムな名前を作成する / Create a name from random parts.
- * @param out_string 作成した名を保管する参照ポインタ
+ * @return std::string 作成した名前で
```

Comment:
"作成した名前" is correct, "で" is not necessary.

Reply:
The forced push should have adopted your suggestion rather than machine-translated result in the return value comments for the four changed functions in that file.